pkg/tsdb/cloudwatch/cloudwatch.go

```diff
@@ -496,9 +496,6 @@ func parseQuery(model *simplejson.Json) (*CloudWatchQuery, error) {
 	}
 
 	alias := model.Get("alias").MustString()
-	if alias == "" {
-		alias = "{{metric}}_{{stat}}"
```

Comment:
Need to set default alias in somewhere.

Please check https://github.com/grafana/grafana/pull/9752 .

Reply:
I don't think we need a default value as when there is no alias defined, we are displaying the query as it is and handling the case on line 524. By assigning a default value, we are adding a complexity just for the sake of assigning a default value to a variable. By default in go, string values are empty string so we handle that in the else condition on line 524. @bergquist - Any suggestions?

Reply:
@mtanda Any comments?

Reply:
The current behavior is showing some legend defined by default alias setting. Incompatibility will cause confusion, it should announce in the release note.

I can't judge which behavior is better and don't have a strong opinion.